### DISCUSSION TITLE
Botman button add url, class, id, or function

### DISCUSSION THREAD

**Discussion by aleresoftware** [botman/botman#1350] [Q&A]
Hi,
I have been trying to add url in the button like this way
   $question->addButtons([
                 Button::create('test')
                     ->additionalParameters([
                         'url' => 'https://www.google.com/',
                     ])
             ]);
But it does not work. Is there any alternative way to put url in the button.
Same like, I am wondering if we can add class or id in the button also

**Comment by filippotoso** [comment]
Which driver are you using?

**Reply by aleresoftware** [reply]
I am working without using driver. So basically using botman to create chatbot for my own website

**Reply by filippotoso** [reply]
So you are using the web driver (https://botman.io/2.0/driver-web) and have installed the web widget (https://botman.io/2.0/web-widget) on your website, right?

**Reply by aleresoftware** [reply]
yes, without using Botman studio

**Reply by aleresoftware** [reply]
Hi Filippotoso,
can you please help me out to figure out the issue?
